scripts/gofmtcheck.sh

```diff
@@ -2,12 +2,12 @@
 
 # Check gofmt
 echo "==> Checking that code complies with gofmt requirements..."
-gofmt_files=$(gofmt -l `find . -name '*.go' | grep -v vendor`)
```

Comment:
This is actually a script that is present in all the terraform provider repos. So I don't really feel the need to make changes to it.

Reply:
I agree, but Backticks must die.

Reply:
Why must backticks die?

Reply:
Well `die` is a strong word, but especially in cases where we are nesting bash executions $() is preferred to backticks. Its more deterministic execution, and one does not have to worry about escaping and such. Ref http://mywiki.wooledge.org/BashFAQ/082